lib/internal/util.js

```diff
@@ -876,7 +877,13 @@ function getCIDR(address, netmask, family) {
 }
 
 const handleTypes = ['TCP', 'TTY', 'UDP', 'FILE', 'PIPE', 'UNKNOWN'];
+setOwnProperty(handleTypes, -1, null);
```

Comment:
this feels almost a bit too clever. Why not simply have `_guessHandleType(fd)` return null explicitly? then your return value below could be `return handletypes[type] || type`.